config/internal_router/gen_keys.sh

```diff
@@ -5,8 +5,10 @@ INTERNAL_NETWORK="192.168.0.0/20"
 TEAM_IP_PREFIX="192.168.1."
 ROUTER_ADDRESS="192.168.0.2/20"
 ENGINE_ADDRESS="192.168.1.0/32"
+ELK_ADDRESS="192.168.1.1/32"
```

Comment:
You could simply put that into a whole new subnet, currently the whole `192.168.0.0/20` is intended for internal stuff, so either assign ELK `192.168.0.3/32` or something from the `192.168.2.0/24` network. I would prefer having the checker ids mapped to the last byte of the IP address in the `192.168.1.0/24` network.

Reply:
changed that